doc/development/api/virtual_schema_api.md

```diff
@@ -1214,3 +1214,49 @@ Notes:
 * `distinct`: Optional. Requires set-function capability `GROUP_CONCAT_DISTINCT.`
 * `orderBy`: Optional. The requested order-by clause, a list of `order_by_element` elements. The field `expression` contains the expression to order by. The `group by` clause of a `SELECT` query uses the same `order_by_element` element type. The clause requires the set-function capability `GROUP_CONCAT_ORDER_BY`.
 * `separator`: Optional. Requires set-function capability `GROUP_CONCAT_SEPARATOR`.
+
+##### LISTAGG
+
+`LISTAGG([DISTINCT] arg1[, separator] ON OVERFLOW {ERROR | TRUNCATE [truncationFiller] {WITH | WITHOUT} COUNT}) [WITHIN GROUP (orderBy)]`
+ (requires set-function capability LISTAGG)
+
+```json
+{
+    "type": "function_aggregate_listagg",
+    "name": "LISTAGG",
+    "distinct": true,
+    "arguments": [
+    {
+        ...
+    }
+    ],
+    "separator": ", ",
+    "overflowBehavior":
+    {
+        "type": "TRUNCATE",
+        "truncationType": "WITH COUNT",
+        "truncationFiller": "..."
+    },
+    "orderBy": [
+        {
+            "type": "order_by_element",
+            "expression":
+            {
+              "type": "column",
+               "columnNr": 1,
+                "name": "USER_ID",
+                "tableName": "CLICKS"
+            },
+            "isAscending": true,
+            "nullsLast": true
+        }
+    ]
+}
+```
+
+Notes:
+
+* `distinct`: Optional. Requires set-function capability `LISTAGG_DISTINCT`.
+* `separator`: Optional. Requires set-function capability `LISTAGG_SEPARATOR`.
+* `overflowBehavior`: type is `ERROR` (requires set-function capability `LISTAGG_ON_OVERFLOW_ERROR`) or `TRUNCATE` (requires set-function capability `LISTAGG_ON_OVERFLOW_TRUNCATE`). Only for `TRUNCATE` the members `truncationType` and optionally `truncationFiller` exist. `truncationType` is `WITH COUNT` or `WITHOUT COUNT`.
```

Comment:
`type`:
```suggestion
* `overflowBehavior`: `type` is `ERROR` (requires set-function capability `LISTAGG_ON_OVERFLOW_ERROR`) or `TRUNCATE` (requires set-function capability `LISTAGG_ON_OVERFLOW_TRUNCATE`). Only for `TRUNCATE` the members `truncationType` and optionally `truncationFiller` exist. `truncationType` is `WITH COUNT` or `WITHOUT COUNT`.
```
Also, for IS JSON and JSON_VALUE we included the double quotes like this:
```suggestion
* `overflowBehavior`: `type` is `"ERROR"` (requires set-function capability `LISTAGG_ON_OVERFLOW_ERROR`) or `"TRUNCATE"` (requires set-function capability `LISTAGG_ON_OVERFLOW_TRUNCATE`). Only for `"TRUNCATE"` the members `truncationType` and optionally `truncationFiller` exist. `truncationType` is `"WITH COUNT"` or `"WITHOUT COUNT"`.
```
Should we do the same here? Or adapt the JSON description?

Reply:
It's okay to make it similar to the JSON. We can change the formatting later